modular_skyrat/modules/xenos_skyrat_redo/code/xeno_types/spitter.dm

```diff
@@ -0,0 +1,78 @@
+/// SKYRAT MODULE SKYRAT_XENO_REDO
+
+/mob/living/carbon/alien/humanoid/skyrat/spitter
```

Comment:
no boiler or prae evo?

Reply:
prae is next up from drone to get to queen, like tg's evo path, boiler was a maybe for me, but we don't REALLY have any long open areas with exception of main halls for them to be useful.